### DISCUSSION TITLE
Some things i need help with

### DISCUSSION THREAD

**Discussion by Loofue** [SirBlobman/CombatLogX#818] [Q&A]
1. How do i change it so people dont glow in combat
2. How do i change it so people can mine in combat
3. How do i change it so people can open their inv in combat

**Comment by ghost** [comment]
for the 1st one remove glowing.jar from /plugins/combatlogx/expansions

**Comment by ghost** [comment]
i think the other stuff u can turn off in the cheat prevention expansion config

**Comment by SirBlobman** [comment]
Yes, @Cooperzilla is correct. Remove the glowing expansion and configure your CheatPrevention expansion.
